.pre-commit-config.yaml

```diff
@@ -23,3 +23,4 @@ repos:
     hooks:
       - id: trailing-whitespace
         exclude: (^dataprofiler/tests/data/|^dataprofiler/tests/speed_tests/data/)
+      - id: debug-statements
```

Comment:
@jakleh did you run pre-commit with this new `id`? Just confirming there are no changes to the repo with this addition

Reply:
excellent